cmd/garm-cli/cmd/common.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"github.com/cloudbase/garm/client/credentials"
+	"github.com/cloudbase/garm/params"
+)
+
+func queryForgeType(credentialName string) error {
+	if forgeType == "" {
+		// TODO do this on server?, where we can query by Name
+		list, err := apiCli.Credentials.ListGiteaCredentials(credentials.NewListGiteaCredentialsParams(), authToken)
```

Comment:
Gitea and Github credentials each have their own model, so for consistency, we would have to query both and see which one has the creds with that name. If both have identically named creds, we would need to err out and ask for the --forge-type.

I think this would be more efficient if implemented server side: https://github.com/cloudbase/garm/blob/18ef27bb9138f49f861171e6db14ca13c85910c1/runner/organizations.go#L32-L51

It would save the 2 extra API calls. If we omit the forge type, we could use the default branch of the switch to query both models. Right now, `param.GetForgeType()` defaults to github for backwards compatibility.

If we do this server side, then the client can just drop the default forge type and leave the field empty.